tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -7,6 +7,8 @@ Define {
 	DEEP_BUFFER_PIPELINE_SRC	'mixin.15.1'
 	DEEP_BUFFER_PIPELINE_SINK	'mixout.2.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer HDA Analog'
+        HDA_ANALOG_CAPTURE_RATE         48000
+        HDA_ANALOG_PLAYBACK_RATE        48000
```

Comment:
just ask a question here: ACE platform should also have HDA config platform, do we need change ACE platform as well?

Reply:
I think we use this same .conf for ACE too. The naming is confusing.

Reply:
agree, if also use this, we may need change name and line 9 to remove HDA config

Reply:
I hope the filenames change for ACE could be other PR. There's quite many cavs-*.conf files.

Maybe the line numbers no more match. What's the issue with HDA config?

Reply:
if the topology does not have NHLT blob then the filename can be the same.
In any case the development topologies should not be copied to the production directory (I just did that on my setup, so don't follow my lead).
If these worked on ACE then there is no NHLT or the NHLT is not used from topology, it can be ripped out?

My topology change PR is not touching development only creates a production target and merges the avs-tplg and sof-ace-tplg under it.

Reply:
I don't think there's 96 kHz DMIC blobs in BIOS so it's safest to use add the blob to topology.

Reply:
@singalsu, my point is: you cannot use the same NHLT blob on CAVS2.5 and ACE1/2, so if the NHLT is taken from the topology file then you must have different files. This is nothing new, I just pointed out that it looks to me that shis change is only supporting CAVS2.5 and the generic once needs namespacing.